Resources/states.geojson

```diff
@@ -0,0 +1,117 @@
+{
+  "type": "FeatureCollection",
```

Comment:
Consider using git lfs to track this file type (geojson) in the future. The more we can leverage git lfs for data files, the more we gain in terms of speed in pushing and pulling changes from GitHub.

Reply:
No need to do anything now with this.